gammapy/catalog/gammacat.py

```diff
@@ -54,25 +54,246 @@ class SourceCatalogObjectGammaCat(SourceCatalogObject):
+
+        ss += '\n{:<15s} : {}\n'.format('TGeVCat ID', d['tgevcat_id'])
+        ss += '{:<15s} : {}\n'.format('TGeVCat name', d['tgevcat_name'])
+
+        return ss
+
+    def _info_position(self):
+        """Print position info."""
         d = self.data
+        ss = '\n*** Position info ***\n\n'
+
+        ss += 'SIMBAD:\n'
+        ss += '{:<20s} : {:.3f}\n'.format('RA', d['ra'])
+        ss += '{:<20s} : {:.3f}\n'.format('DEC', d['dec'])
+        ss += '{:<20s} : {:.3f}\n'.format('GLON', d['glon'])
+        ss += '{:<20s} : {:.3f}\n'.format('GLAT', d['glat'])
 
-        ss = 'Source: {}\n'.format(d['common_name'])
-        ss += 'source_id: {}\n'.format(d['source_id'])
-        ss += 'reference_id: {}\n'.format(d['reference_id'])
-        ss += '\n'
+        ss += '\nMeasurement:\n'
+        ss += '{:<20s} : {:.3f}\n'.format('RA', d['pos_ra'])
+        ss += '{:<20s} : {:.3f}\n'.format('DEC', d['pos_dec'])
+        ss += '{:<20s} : {:.3f}\n'.format('GLON', d['pos_glon'])
+        ss += '{:<20s} : {:.3f}\n'.format('GLAT', d['pos_glat'])
+        ss += '{:<20s} : {:.3f}\n'.format('Position error', d['pos_err'])
 
-        ss += 'RA          : {:.2f}\n'.format(d['ra'])
-        ss += 'DEC         : {:.2f}\n'.format(d['dec'])
-        ss += 'GLON        : {:.2f}\n'.format(d['glon'])
-        ss += 'GLAT        : {:.2f}\n'.format(d['glat'])
-        ss += '\n'
         return ss
 
-    def info(self):
-        """Print summary info."""
-        print(self)
+    def _info_morph(self):
+        """Print morphology info."""
+        ss = '\n*** Morphology info ***\n\n'
+        d = self.data
+        ss += '{:<25s} : {}\n'.format('Morphology model type', d['morph_type'])
+        ss += '{:<25s} : {:.3f}\n'.format('Sigma', d['morph_sigma'])
+        ss += '{:<25s} : {:.3f}\n'.format('Sigma error', d['morph_sigma_err'])
+        ss += '{:<25s} : {:.3f}\n'.format('Sigma2', d['morph_sigma2'])
+        ss += '{:<25s} : {:.3f}\n'.format('Sigma2 error', d['morph_sigma2_err'])
+        # Call these major/minor axis instead?
+        # Signify (gauss) for Sigma above?
+
+        ss += '{:<25s} : {:.3f}\n'.format('Position angle', d['morph_pa'])
+        ss += '{:<25s} : {:.3f}\n'.format('Position angle error', d['morph_pa_err'])
+        ss += '{:<25s} : {}\n'.format('Position angle frame', d['morph_pa_frame'])
+
+        return ss
+
+
+    def _info_spectral_fit(self):
+        """Print spectral info."""
+        d = self.data
+        ss = '\n*** Spectral info ***\n\n'
+        ss += '{:<32s} : {:.3f}\n'.format('Significance (*energy range*)', d['significance'])
+        ss += '{:<32s} : {:.3f}\n'.format('Livetime', d['livetime'])
+
+        spec = d['spec_type']
+        str = ''
+        if(spec == 'pl2'):
+            str = '(integral power law)'
+        ss += '\n{:<15s} : {} {}\n'.format('Spectrum type', spec, str)
+
+        # Spectral model parameters
+        if(spec == 'pl'):
```

Comment:
You don't need parens around expressions in the if statenemts. Remove.
Also: use the if - elif - elif - else pattern ending with:
```
else:
    raise ValueError('Spectral model printout not implemented: {}'.format(spec))
```

Long-term, it might be better to instantiate spectral model objects and just print out those.
But for now, what you have here is find I think (I don't remember what we do for the other catalogs to print out the spectral models at the moment).